components/02-molecules/video/_yds-video.scss

```diff
@@ -0,0 +1,22 @@
+@use '../../00-tokens/tokens';
+
+.video {
+  @include tokens.spacing-page-section;
+}
+
+.video__inner {
+  display: grid;
+  gap: var(--size-spacing-3);
+}
+
+.video__heading {
+  margin-bottom: var(--size-spacing-1);
+}
+
+.video__text {
+  font: var(--font-style-body-s);
```

Comment:
Sorry! I should have caught this before... all typography should use one of the mixins we have. So, in this case, instead of 
```
font: var(--font-style-body-s);
```
you should do
```
@include tokens.body-s;
```

You can see all of the possible options when you toggle the "Font details" under one of the body styles here: https://deploy-preview-145--dev-component-library-twig.netlify.app/?path=/story/tokens-typography--body-styles (The heading ones have it too)

This is because some of the styles (really just the headings at this point, but we want to keep implementation consistent) have things like letter-spacing or text-transforms applied, and that can't be captured as a part of a single css custom property